tests/5.0/requires/test_requires_unified_address.c

```diff
@@ -20,23 +20,42 @@ int unified_address() {
 
    int errors = 0;
    int i;
-   int * mem_ptr = (int *)malloc(N * sizeof(int));
+   int *mem_ptr = (int *)omp_target_alloc(N * sizeof(int), omp_get_default_device());
+   int *mem_ptr2 = NULL;
 
    OMPVV_ERROR_IF(mem_ptr == NULL, "Memory was not properly allocated");
```

Comment:
I suggest to return from the function after `OMPVV_ERROR_IF`: `if(mem_ptr == NULL) return 1;`